sentry/templates/configmap-snuba.yaml

```diff
@@ -44,8 +44,8 @@ data:
             "errors_v2_ro",
             "profiles",
         },
+        "single_node": {{ .Values.externalClickhouse.singleNode | quote }},
```

Comment:
Shouldn't the value be `True` or `False` instead of a string?

Reply:
Thanks! Changed